src/generators/generator.js

```diff
@@ -17,7 +17,7 @@ class Generator {
     this.componentName    = args.componentName;
     this.templatePath     = args.templatePath;
     this.testTemplatePath = args.testTemplatePath;
-
+    this.wrapFilesInFolders = args.wrapFilesInFolders;
     // project wide settings
     this.sourceBase    = args.settings.getSetting('sourceBase');
     this.testBase      = args.settings.getSetting('testBase');
```

Comment:
True, I was thinking that there might be some cases in the future where you want it on an individual level, but maybe thats a premature notion. 

EG: I want my dumb components in a single file, smart components in a folder, and redux modules not in a folder. Obviously it makes more sense to implement that when necessary, so I am switching it back.